Gemfile

```diff
@@ -76,6 +76,7 @@ gem 'uglifier', '>= 1.3.0'
 # as the front-end framework
 gem 'autoprefixer-rails'
 gem 'bootstrap-sass', '~> 3.4.0'
+gem 'bootstrap-tagsinput-rails'
```

Comment:
@Ana06 I was not able to reuse the user_selector_input to allow user to enter email id and if it is a valid email then only it was converting to tags(while creating a booth) so that user can differentiate easily between emails.

Reply:
@Ana06 How can I use user_selector_input when I am not looking for users, I am just typing new unregistered email ids to invite.

Reply:
I believe Ana actually meant that you can use selectize - did you search about using it without searcing through existing data? Is it possible?

Reply:
@differentreality I was working on it. Yes it is possible. After Ana suggested it, I was trying to use that. I will update the PR.